### PR TITLE
metrics: add IME thread cpu panel

### PR DESCRIPTION
**Pull request by overvenus**
### What is changed and how it works?

Issue Number: ref #16141 

{92A904BB-7934-46A3-B7A8-976F01A1A017}

What's Changed:

```commit-message

```

### Check List

Tests 

- No code

### Release note

```release-note
None
```

### CODE REVIEW

File: metrics/grafana/tikv_details.dashboard.py [hunk 1]
```diff
@@ -1444,6 +1444,25 @@ def ThreadCPU() -> RowPanel:
...
+                description="The CPU utilization of IME threads",
+                yaxes=yaxes(left_format=UNITS.PERCENT_UNIT),
+                targets=[
+                    target(
+                        expr=expr_sum_rate(
+                            "tikv_thread_cpu_seconds_total",
+                            label_selectors=['name=~"ime.*"'],
+                            by_labels=["instance", "name"],
+                        ),
+                        legend_format="{{instance}}-{{name}}",
```
Comment (glorv) [inline metrics/grafana/tikv_details.dashboard.py:1460]: Shall we only display a total cpu by default and hide the details cpu usage per-type?

Comment (overvenus) [reply metrics/grafana/tikv_details.dashboard.py:1460]: Since the dashboard named "tikv-details", let's keep the details.

Comment (glorv) [reply metrics/grafana/tikv_details.dashboard.py:1460]: Based on the fact that IME will start quite a number of threads and most of them only consume very little cpu time in most time(if not all), I wonder what is the benefit of show these noise-like data by default.

Comment (overvenus) [reply metrics/grafana/tikv_details.dashboard.py:1468]: Done.

**Review by glorv — APPROVED** [review]

**Review by SpadeA-Tang — APPROVED** [review]

**Pull request merged**